src/adapters/adapter.js

```diff
@@ -163,4 +181,13 @@ export default class Adapter extends Configurable {
                     {[this.name]: adapterUserId});
   }
 
+  promptUser = (data, cb) => {
+    this.bot.emitter.emit(AO.WHISPER_USER,this.name, data);
+    if (this.prompts[data.userId]) {
+      this.prompts[data.userId].push({data, cb});
```

Comment:
spaces fo' yo' braces (here and below). also a space after AO.WHISPER_USER above.